### ISSUE TITLE
HTML version of Survey Consent Form

### ISSUE THREAD

**Issue by emily-phet** [phetsims/tasks#874]
We are creating a survey that requires the inclusion of a consent form (attached). The consent form needs to be screen reader accessible. We'd like to include a link in the survey to an HTML page that includes the consent information - this will allow the consent information to be screen reader accessible without having to have lengthy text in the survey question itself.  

Can someone create an HTML page that has the look and content of this consent form by the end of this week (June 30th)?
[12-0666_consent_form__25oct12_.docx](https://github.com/phetsims/tasks/files/1105469/12-0666_consent_form__25oct12_.docx)

**Comment by ariel-phet** [comment]
@mattpen is this something you could do quickly? Feel free to recruit @zepumph if it is time consuming and could use some help later in the week

**Comment by mattpen** [comment]
This should be doable.  @ariel-phet Does it need to be published on phet.colorado.edu by the end of the week?  There are some things in master that are in QA testing at the moment, but I could put up a static version of the file as a workaround.

**Comment by mattpen** [comment]
@emily-phet - I noticed there is a spot for a link in the description, should I post a pdf version of this file along side it?

Here is the text in question:

> A pdf copy of this consent form can be obtained from the linked file (click here).

I'm planning to publish the consent form as https://phet.colorado.edu/consent-form-educator-surveys.html.  I could publish the pdf version as https://phet.colorado.edu/consent-form-educator-surveys.pdf.

**Comment by mattpen** [comment]
I added the consent form in this commit: https://github.com/phetsims/website/commit/160ca630f0aa705a09ba4acae28e2cd161d650ba

**Comment by mattpen** [comment]
FYI - the consent form is now available for preview at https://phet.colorado.edu/consent-form-educator-surveys.html

**Comment by mattpen** [comment]
I tested the form with NVDA and it seems to work OK, although I'm not an expert user.

**Comment by ariel-phet** [comment]
Unassigning myself, thanks for the work @mattpen!

**Comment by emily-phet** [comment]
@mattpen - this looks great! Can you please create the pdf version, and add the link (the one you mentioned above sounds perfect). Also, can you add a link at the top that reads "Return to survey". I'm not sure yet what the survey link will be...so this will need to be a placeholder for now.

**Comment by emily-phet** [comment]
@mattpen - Not sure when you're working this week, but if you could give me a heads up when you'll be able to get to this, that would be helpful for me. Thanks!

**Comment by mattpen** [comment]
@emily-phet - I can work on this today.  Just let me know when you have the link and I can post it as well.

**Comment by mattpen** [comment]
@emily-phet - I added the link and the PDF document to the main site, available at the links above.  Right now the link just returns to the consent form.  It should be easy to add the link once you have it.

**Comment by emily-phet** [comment]
@mattpen Thanks, everything looks great. 

Here's the survey link: https://cuboulder.qualtrics.com/jfe/form/SV_2rjPNaEnZ1TTEj3 

Please let me know when this is up.

**Comment by mattpen** [comment]
@emily-phet - The link is up now.  It's not working in the PDF, although it looks like that is intended for the "printable format" link.  Should I remove the return to survey link from the PDF? Leave it as is? Or invest some time into getting it to work?

**Comment by emily-phet** [comment]
@mattpen Great, thanks! If it's easy to remove the "return to survey" part in the pdf, go ahead. If it's difficult, just leave it in...personally I didn't expect it to work since I was landing on a pdf version and it doesn't look like an active link. Otherwise, we're all set, thanks!

**Comment by mattpen** [comment]
@emily-phet - Thanks, I removed the link from the pdf and republished it.
